gc/gc.go

```diff
@@ -33,7 +33,7 @@ func collectService(svc icinga2.Service, c config.Configuration, downtimes []ici
 	l := c.GetLogger()
 	icinga := c.GetIcingaClient()
 
-	_, heartbeat := svc.Vars["label_heartbeat"]
+	heartbeat := svc.LabelHeartbeat
```

Comment:
As noted in https://github.com/vshn/go-icinga2-client/pull/10, please provide a replacement API to access a service's custom variables, so we don't need to add a bunch of Signalilo-specific fields to the generic `Service` type.